### PR TITLE
feat: collapsible menu

### LINKED ISSUE #207: proposition: add collapsible capability to side menu
side menu can be collapsed on a way only icons are visible
here is a menu in `expanded` state with collapse toggle below
image
here is collapsed state:
image

### PR DESCRIPTION
**Pull request by shootermv**
# Description
at desktop, side menu has `collapsible menu` toggle at its  bottom
closes #207 

## screenshots
expanded mode:
image
collapsed mode:
image

### CODE REVIEW

**Review by NoamGaash — APPROVED** [review]

File: src/routes/index.tsx [hunk 1]
```diff
@@ -25,58 +25,58 @@ export const PAGES = [
   {
     label: TEXT_KEYS.dashboard_page_title,
     path: '/dashboard',
-    icon: LaptopOutlined,
```
[the remark this answers is not in the document]
Comment (shootermv) [reply src/routes/index.tsx:29]: how to check if there is performance problem?

File: tests/about.spec.ts [hunk 2]
```diff
@@ -4,8 +4,8 @@ test.describe('About Page Tests', () => {
     await page.goto('/')
     await page.getByText('אודות').click()
     await expect(page).toHaveURL('http://localhost:3000/about')
-    const locator = await page.getByText('אודות')
-    await expect(locator).toHaveClass('menu-item active')
+    const locator = await page.locator('li:has-text("אודות")')
```
[the remark this answers is not in the document]
Comment (shootermv) [reply tests/about.spec.ts:7]: the html tags of menu items changed
now the `active` class is on li tag, although  the `a` inner element has a text

File: src/routes/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (NoamGaash) [reply src/routes/index.tsx:29]: no,
feel free to merge it.

**Comment by xoRmalka** [comment]
Need to set the width to 220px, also after the app is rendered the opening/ closing of the sider is very slow :)

**Comment by NoamGaash** [comment]
@xoRmalka @shootermv it's already merge, so feel pree to open issue or another PR if needed

**Pull request merged**